app/controllers/api/concerns/act_as_api_request.rb

```diff
@@ -4,7 +4,6 @@ module ActAsApiRequest
       extend ActiveSupport::Concern
 
       included do
-        skip_before_action :verify_authenticity_token
```

Comment:
are you sure we can just remove this line? Wouldn't it cause our frontend apps start getting `ActionController::InvalidAuthenticityToken` when submitting forms?

Reply:
Since my point of view, If you access by API each request is independent of each other, and the uniq way to validate this (isolate) request is sending the correct information in the HEADER that not include 'authenticity_token'.

I think that by removing this line ***we should not have problems from ```ActionController::InvalidAuthenticityToken```***

I imagine two scenarios to use the API:

- You have completely separated the back from the front, like two independent systems: in this case, you have no way to send an authenticity_token from the front form, and no problem with that if you send the correct token in each request

- You have a rails project with the back and the front together: Ok, in this case Yes; remove the line  ```skip_before_action :verify_authenticity_token``` could be a problem if in your front you use a form but; if you have in the same project the back & front together your forms should not (although they could ) call a request with the prefix prefix```/api/v1/{anything}``` .
Conceptually, if you call with the ```api/v1/...``` prefix you must validate only by token.
And if you have this last case, then the forms should call the internal controllers without the API prefix and in them we should put the line ```before_action :verify_authenticity_token```. 
As I said before, it's just a point of view

Reply:
Something to add @santib 

Looking at the [route.rb](https://github.com/rootstrap/rails_api_base/blob/master/config/routes.rb) file, can we ensure that no route from `/admin` through by `/api/v1` ?

Reply:
yep, you are right